test/e2e/testdata/backend-tls-settings.yaml

```diff
@@ -6,52 +6,73 @@ metadata:
 spec:
   gatewayClassName: "{GATEWAY_CLASS_NAME}"
   listeners:
-    - name: https
+  - name: http
+    port: 80
+    protocol: HTTP
+---
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: http-with-backend-tls
+  namespace: gateway-conformance-infra
+spec:
+  parentRefs:
+  - name: backend-namespaces
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /backend-tls
+    backendRefs:
+    - name: tls-backend
       port: 443
-      protocol: HTTPS
```

Comment:
Client TLS is not required for the Backend TLS setting test. Removing it to make the test simpler and easier to understand.